addons/mail/static/src/core/thread_model.js

```diff
@@ -160,6 +162,10 @@ export class Thread {
         return ["chat", "group"].includes(this.type);
     }
 
+    get isChannelGroup() {
+        return ["channel", "group"].includes(this.type);
+    }
```

Comment:
Misleading name for this getter. The getter is not worth it, so if you drop it then you don't have to name it